source/_integrations/comelit.markdown

```diff
@@ -0,0 +1,27 @@
+---
+title: Comelit SimpleHome
+description: Instructions on how to integrate Comelit SimpleHome home automation devices into Home Assistant.
+ha_category:
+  - Light
+ha_release: '2023.09'
```

Comment:
I think that `2023.9` is a default format.

Reply:
Thx, fixed